src/widgets/_emojipicker.scss

```
@@ -0,0 +1,58 @@
+.emoji-picker {
+    entry {
+        margin: rem(12px);
+        margin-bottom: 0;
+    }
+
+    // Emoji are labels too, so we have to be specific
+    box.vertical > label {
+        font-weight: 700;
+        //Intentionally off-by-one
+        margin-left: rem(7px);
+        margin-right: rem(7px);
+        opacity: 0.8;
+    }
```

If you're trying to match `.h4` styling, just extend it to ensure it stays in sync.

```suggestion
    box.vertical > label {
        @extend .h4;

        // Intentionally off-by-one
        margin-left: rem(7px);
        margin-right: rem(7px);
    }
```

---

woah didn't know you could do that. That is extremely useful